docs/content/modules/tensor_mechanics/systems/Materials/HyperbolicViscoplasticityStressUpdate.md

```diff
@@ -1,10 +1,10 @@
-#RecomputeRadialReturnHyperbolicViscoplasticity
-!description /Materials/RecomputeRadialReturnHyperbolicViscoplasticity
+#HyperbolicViscoplasticityStressUpdate
+!description /Materials/HyperbolicViscoplasticityStressUpdate
 
-!devel /Materials/RecomputeRadialReturnHyperbolicViscoplasticity float=right width=auto margin=20px padding=20px background-color=#F8F8F8
+!devel /Materials/HyperbolicViscoplasticityStressUpdate float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
 ## Description
-{!content/modules/tensor_mechanics/common_documentation/supplementalRecomputeRadialReturn.md!}
+{!content/modules/tensor_mechanics/common_documentation/supplementalRadialReturnStressUpdate.md!}
```

Comment:
This allows me to duplicate common documentation on several pages without writing the same information multiple times; in this case it's a brief intro to J2 radial return and a picture of a yield surface - I like the option to have this information on the same page rather than a link away